### ISSUE TITLE
Differentiate the display parameters

### ISSUE THREAD

**Issue by Bindou01** [jurialmunkey/skin.arctic.fuse#681]
### Skin section

Video OSD

### Feature description

Differentiate the display parameters of the player's OSD depending on whether you fast forward or press OK.

### Why is this feature needed? Please justify and explain.

This would reduce the amount of information and the duration of display of the progress bar when you are just moving forward.  While allowing a longer and more complete display of the OSD when OK is pressed.

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
You can press info on a remote to see more detail.

There is also an info button in the controls.

**Comment by Bindou01** [comment]
> You can press info on a remote to see more detail.
> 
> There is also an info button in the controls.

Good idea, but you know, the more modern televisions are, the fewer buttons their remote controls have.  ^^ In order to maximize the compatibility of the Kodi/Fuse interface, I think it is necessary to think about ergonomics like Netflix and co.  with a minimum of programmable buttons.  I don't think I know anyone anymore who doesn't press OK to display the information.  ^^ For example, with the Estuary base skin, the Clear Logo only appears if you click OK.  Not if we fast forward.  I would find it better, especially with Fuse where the Clear Logo display takes up a lot of space on the screen.  Also, this doesn't really solve the timing issue.  I think that the appearance time required for fast forwarding is not the same as for navigating the player's OSD to modify a soundtrack for example, or to read other more complex information.  I don't know if I'm explaining myself well.

**Comment by jurialmunkey** [comment]
There is a button in the osd controls to show full info. You do not need a button on your remote.

**Comment by Bindou01** [comment]
> There is a button in the osd controls to show full info. You do not need a button on your remote.

Ah yes I see what you mean.  I actually use it to display information relating to production.  But I thought it was not bad to at least have the title of the film appearing somewhere without having to go into more information.  However, perhaps it would be enough to place the Clear Logo elsewhere so that it takes up less space on the screen and is less distracting when fast forwarding and looking for a specific scene.  I don't know.  Differentiating the settings was an idea, but maybe there are other solutions.

**Comment by Bindou01** [comment]
Also, I notice that even if we deactivate the Clear Logo, the shadow zone remains the same.  I think it's a bit awkward when navigating through a movie to search for a scene.

**Comment by Bindou01** [comment]
Finally, I don't know if I should write it here or create a ticket again.  I notice that when we advance by chapter, at the top, we do not see chapter information evolving, but time information.

**Comment by xyzfre** [comment]
> > There is a button in the osd controls to show full info. You do not need a button on your remote.
> 
> 
> 
> Ah yes I see what you mean.  I actually use it to display information relating to production.  But I thought it was not bad to at least have the title of the film appearing somewhere without having to go into more information.  However, perhaps it would be enough to place the Clear Logo elsewhere so that it takes up less space on the screen and is less distracting when fast forwarding and looking for a specific scene.  I don't know.  Differentiating the settings was an idea, but maybe there are other solutions.

Have you tried in skin settings behavior there are some options there for the OSD screen to accommodate what info is displayed on the the screen which right now the settings are GREAT.

**Comment by Bindou01** [comment]
affirmative

**Comment by jurialmunkey** [comment]
> Finally, I don't know if I should write it here or create a ticket again. I notice that when we advance by chapter, at the top, we do not see chapter information evolving, but time information.

Separate bugs or request should be reported in separate issues.

Random comments in the middle of vaguely related threads are very difficult for me to keep track of. It's like giving me a trash bag full of random loose documents and expecting me to organise them all. 

Separate issues for different bugs or features make it much more organised and easier for me to track than one issue going on several different tangents.
